### PR TITLE
search with casestudies and products

### PR DESCRIPTION
**Pull request by djrobstep**

### CODE REVIEW

File: app/main/views/search.py [hunk 1]
```diff
@@ -54,6 +61,26 @@ def get_all_domains(data_api_client):
     return [_['name'] for _ in data_api_client.req.get_domains()['domains']]
 
 
+SELLER_TYPES = od([
+    ('Start up', 'start_up'),
```
Comment (matt-sm) [inline app/main/views/search.py]: should these be hardcoded?

File: app/main/views/search.py [hunk 2]
```diff
@@ -203,29 +277,75 @@ def supplier_search():
...
-
-    seller_type_filters = {st: st in selected_seller_types for st in SELLER_TYPE_KEYS}
+    seller_type_filters = {st: to_seller_type_key(st) in selected_seller_types_keys for st in SELLER_TYPES.keys()}
 
     sidepanel_roles = [Option('role', role, role, role in selected_roles) for role in roles]
     sidepanel_filters = [
         Filter('Capabilities', sidepanel_roles),
     ]
 
+    PRODUCTS = True
```
Comment (matt-sm) [inline app/main/views/search.py]: remove?

**Comment by matt-sm** [comment]
no tests?

**Comment by djrobstep** [comment]
tests forthcoming

File: app/main/views/search.py (hunk 2 above)
Comment (djrobstep) [inline app/main/views/search.py]: yep will remove

File: app/main/views/search.py (hunk 1 above)
Comment (djrobstep) [inline app/main/views/search.py]: They need to be defined somewhere. Could move them to the database?

**Pull request merged**